docs/quickstart.md

```diff
@@ -21,37 +21,47 @@ docker run -p 30001:30001 -p 30002:30002 -p 30003:30003 redislabs/rgcluster:late
 
 ## Building
 
-### Clone
-To get the code and its submodules do the following:
+### Prerequisites
+* Install build-essential:
+
 ```
-git clone https://github.com/RedisGears/RedisGears.git
-cd RedisGears
-git submodule update --init --recursive
+sudo apt-get install build-essential
```

Comment:
@itamarhaber maybe I should remove this as its os specific?

Reply:
@MeirShpilraien `make setup` will install dev tools for any platform. Maybe we should play with order of operations. Also, I can deliver the redis installation with `./deps/readies/bin/getredis` (this requires some tweaking but that's a quick fix).

Reply:
Yes I know, but I thought to split it to redis installation and building RedisGears and I thought it makes sense to first install redis as Prerequisites. What do you think?

Reply:
`system-setup.py` does really belong with the installation. I would have naturally would include the installation of Redis inside it (in fact, it's there for macOS), however, as on some systems it requires a build of it's own, and on other systems the package manager also installs Redis as a service, I felt that separating Redis from the pack was the right thing to do. I would prefer to install Redis in one command/script, and we probably do want to build it ourselves as most older systems would not have Redis 6 in their installer repos.

Reply:
@itamarhaber what do you think?